### ISSUE TITLE
Sync All in History not working

### ISSUE THREAD

**Issue by terryupton** [BrowserSync/UI#45]
When I try and Sync All to a historical URL in the UI, none of the instances of the website update or change? Is this a known bug or anyway to resolve this?

**Comment by foxx** [comment]
I'm having this same problem, any ideas?

**Comment by foxx** [comment]
Upon inspecting the websocket frames, I can see that the browsers receive the instruction... but they don't seem to do anything with it..

```
42/browser-sync,["browser:location", {path: "/", override: true, url: "/"}]
0
:
"browser:location"
1
:
{path: "/", override: true, url: "/"}
override
:
true
path
:
"/"
url
:
"/"
Name
?EIO=3&transport=websocket&sid=5IipwVfzaZa4tZ9CAAAH
```

**Comment by bernhard-appdirect** [comment]
@foxx @terryupton  We had the same issue and found the undocumented solution here: https://github.com/BrowserSync/browser-sync/issues/1363

This also fixed the "Reset All" button.

````JavaScript
ghostMode: { location: true }

**Comment by terryupton** [comment]
@bernhard-appdirect excellent! Thank you very much. This has solved the issue for me.

**Comment by shakyShane** [comment]
This is fixed now in `browser-sync@2.18.10`

https://github.com/BrowserSync/browser-sync/commit/1a0505ca562cb2239ef233b3eca77e29aaad7084

:)
